src/core/lib/iomgr/unix_sockets_posix.cc

```diff
@@ -64,6 +64,34 @@ grpc_error* grpc_resolve_unix_domain_address(const char* name,
   return GRPC_ERROR_NONE;
 }
 
+grpc_error* grpc_resolve_unix_abstract_domain_address(
+    const char* name, grpc_resolved_addresses** addrs) {
```

Comment:
I think the first argument should be of type `absl::string_view` instead of `const char*`.  That way, we'll be able to handle paths with embedded NUL bytes once the URI parser can handle those.